cmd/lora-gateway-bridge/main.go

```diff
@@ -113,6 +113,11 @@ func main() {
 			Usage:  "mqtt server password (optional)",
 			EnvVar: "MQTT_PASSWORD",
 		},
+		cli.StringFlag{
+			Name:   "cafile",
```

Comment:
Maybe it would be better to keep this specific to MQTT, thus `mqtt-cafile` `MQTT_CAFILE`? Or `mqtt-ca-cert` and `MQTT_CA_CERT` (to keep the naming consistent with the LoRa Server / LoRa App Server config flags).